### ISSUE TITLE
Support thread-local storage

### ISSUE THREAD

**Issue by arthaud** [NASA-SW-VnV/ikos#59]
IKOS crashes on the following code:

```cpp
static thread_local int x;

int main() {
  return x;
}
```

```
$ ikos test.cpp
[*] Compiling test.cpp
[*] Running ikos preprocessor
[*] Running ikos analyzer
[!] ikos was compiled in debug mode, the analysis might be slow
[*] Translating LLVM bitcode to AR
ikos-analyzer: [...]/test.pp.bc: error: no debug information for llvm function @_ZTWL1x
ikos: error: a run-time error occured
```

**Comment by arthaud** [comment]
Please keep in mind that `ikos` does not handle multithreaded programs, thus thread-local variables are treated as normal global variables.

**Comment by adegomme** [comment]
Hi,
we are facing a similar issue : 
 missing debug information for llvm function _ZTWN7simgrid6kernel7contextL19smx_current_contextE

It looks like the matches symbol should be starting with _ZTW and not _ZTWL1 (no idea why). Changing this in bundle.cpp allowed to go further.
Best regards

**Comment by arthaud** [comment]
Thanks for the report, fixed in 1995ece7f9c319a8baf54a29ac12f7b4894079c4.
